visualization/settings/visualization_config.yaml

```diff
@@ -0,0 +1,4 @@
+visualization_params:
+  model_prediction:
+    model1: ???
```

Comment:
one model is also supported, right? Should change defaults then? Also what about 3+ models? If not supported, better to add some checks for that, I guess

Reply:
then better to not have this filled in to two models by default? Maybe just keep it empty or remove this config completely (you already have a dataclass, right)?